script.js

```diff
@@ -1,76 +1,121 @@
+        return chooseOrderMessage(player, roll1, roll2);
       }
 
       if (!(input == 1 || input == 2)) {
-        return `Incorrect input!! <br> Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+        return (
+          "Incorrect input!! <br>" + chooseOrderMessage(player, roll1, roll2)
+        );
       }
 
       var playerChoice = input;
       player1num = chosennumber(playerChoice, roll1, roll2);
       console.log("Player1 number", player1num);
       player = 2;
+      roll1 = 0;
       return `Player 1, Your number is ${player1num}! <br> Player 2, please press submit to start!`;
     }
 
     if (player == 2) {
       if (!input) {
-        roll1 = randomNumber();
-        console.log("Player 2", roll1);
-        roll2 = randomNumber();
-        console.log("Player 2", roll2);
-        return `Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+        return chooseOrderMessage(player, roll1, roll2);
       }
 
       if (!(input == 1 || input == 2)) {
-        return `Incorrect input!! <br> Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+        return (
+          "Incorrect input!! <br>" + chooseOrderMessage(player, roll1, roll2)
+        );
       }
 
       playerChoice = input;
       player2num = chosennumber(playerChoice, roll1, roll2);
       console.log("Player2 number", player2num);
+      roll1 = 0;
       section = ACTUALGAME;
       return `Player 2, Your number is ${player2num}! <br> Player 2 please press submit to get results!`;
     }
   }
 
   if (section == ACTUALGAME) {
-    var myOutputValue = `Player 1 you have lost! Please press submit to start again`;
+    player1sum += player1num;
+    player2sum += player2num;
+
+    if (gameMode == NORMAL) {
+      var myOutputValue = `Player 2 is leading! Please press submit to start again`;
 
-    if (player1num > player2num) {
-      myOutputValue = `Player 1, you have won! Please submit to play again`;
+      if (player1sum > player2sum) {
+        myOutputValue = `Player 1 is leading! Please submit to play again`;
+      }
+
+      if (player1sum == player2sum) {
+        myOutputValue = `Its a draw! Please submit to play again`;
+      }
     }
 
-    if (player1num == player2num) {
-      myOutputValue = `Its a draw!`;
+    if (gameMode == LOWESTWINS) {
+      var myOutputValue = `Player 2 is leading! Please press submit to start again`;
+
+      if (player1sum < player2sum) {
+        myOutputValue = `Player 1 is leading! Please submit to play again`;
+      }
+
+      if (player1sum == player2sum) {
+        myOutputValue = `Its a draw! Please submit to play again`;
+      }
```

Comment:
[Comment] Nice way to split between the diff modes (Normal / Lowest Wins). Nice good job